distribution/ddf-common/src/main/resources/etc/startup.properties

```diff
@@ -0,0 +1,10 @@
+# Bundles to be started on startup, with startlevel
+mvn\:org.apache.karaf.features/org.apache.karaf.features.extension/4.1.1 = 1
+mvn\:org.apache.felix/org.apache.felix.metatype/1.1.2 = 5
+mvn\:org.apache.karaf.services/org.apache.karaf.services.eventadmin/4.1.1 = 5
+mvn\:org.ops4j.pax.url/pax-url-aether/2.5.2 = 5
+mvn\:org.ops4j.pax.logging/pax-logging-api/1.9.1 = 8
+mvn\:org.ops4j.pax.logging/pax-logging-log4j2/1.9.1 = 8
+mvn\:ddf.platform.osgi/platform-osgi-configadmin/${project.version} = 10
```

Comment:
For reference, take a look at the `startup.properties` file that gets generated in `etc` when on master. Notice that, normally, this line would look like:
```
mvn\:org.apache.felix/org.apache.felix.configadmin/1.8.14 = 10
```
Using a boot feature won't be good enough. We need to actually replace Felix with our own implementation.

Reply:
I'm assuming the resources plugin does get run because the version directive does get replaced. Comments added.